R/calc_aggregate_counts.R

```diff
@@ -29,6 +29,49 @@
 #' calc_aggregate_counts(state = TRUE, all_dates = TRUE)
 #' @export
 
+create_statewide_counts <- function(dat, all_dates) {
+    ## for state-wide historical data, we create weekly or monthly aggregate data
+    if (all_dates) {
+        dat <- dat %>%
+            mutate(Date = lubridate::floor_date(Date, round_))
```

Comment:
`round_` here is not defined -- I think that's why there's the compile error that Lauren mentioned. Seems like if you just passed on `round_` from the calc_aggregate_counts as a function argument that would be quick. I'd consider naming it something longer like `round_to`. Python has a tradition of naming throwaway variables as `_` so I keep misreading the name and I also think it's a tiny bit more descriptive.